packages/vats/src/localchain.js

```diff
@@ -120,7 +120,7 @@ export const prepareLocalChainAccountKit = (zone, { watch }) =>
          * This is safe, since even if the payment lies about its brand, ERTP
          * will reject spoofed payment objects when depositing into a purse.
          *
-         * @param {Payment<'nat'>} payment
+         * @param {ERef<Payment<'nat'>>} payment
```

Comment:
This seems wrong. What caused the change?

Reply:
attn @turadg

Reply:
IIRC it was a fix for what is better fixed as [7f066cc](https://github.com/Agoric/agoric-sdk/pull/9811/commits/7f066ccac3e5a762498d1d340cef0908380777bd)

I probably thought this change was okay because the function wraps that arg in `E()`, but it also passes it as context to a function that won't accept a promise. That should have errored so I filed https://github.com/Agoric/agoric-sdk/issues/9812

Reply:
This change is still necessary to appease tsc.

Reply:
Please add a TODO or a link to an issue.